tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,34,35,py}-django{18,19,110},isort
+envlist = py{39,310,py}-django{32,42},isort
```

Comment:
maybe consider adding `py311` too? Keep in mind that Django 3.2 does not support that one (https://docs.djangoproject.com/en/4.2/faq/install/#what-python-version-can-i-use-with-django)